PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+Hello,
+
+Thank you for submitting a pull request, that's awesome! Please add some info (if applicable) to give us some context on the PR.
+
+Things to consider before you submit the PR:
+
+* Has `CHANGELOG.md` been updated? (optional)
+* Are tests passing locally?
+* Are the files formatted correctly?
+* Did you add unit tests?
+
+We will review the PR as soon as possible, leave feedback, add a tag, etc. and let you know what's going on.
+
+Cheers!
+
+The App Center team!
+
+## Status
+
+Let us know if this is ready to be merged, you are currently working on it, or if you are currently not working on it.
```

Comment:
I don't think the status should ever be "I'm not working on this".  We should just close it if that's the case.  My suggestion: "Let us know if this is ready to be merged or if you are still working on it."

Reply:
Practically, I haven seen very few cases of community PRs that are large code changes, hence the contributors weren't able to finish their work before submitting. IMO this "status" is not very useful. For the purpose of briefness, I would not add this "status".